docs/usage/network_policies.md

```diff
@@ -0,0 +1,148 @@
+ - into the `kube-system` namespace in the Shoot.
+
+Gardener deployed network policies have names describing their purpose, so this document does not describe each policy in detail.
+## Network policies in the `Shoot` namespace in the `Seed`
+
+The network policies in the `Shoot` namespace in the `Seed` can roughly be grouped into policies required for the control plane components and for logging & monitoring.   
+
+The network policy `deny-all` plays a special role. This policy [denies all ingress and egress traffic](https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-deny-all-ingress-and-all-egress-traffic) from each pod in the `Shoot` namespace.
+So per default, a pod running in the control plane cannot talk to any other pod via IP or indirectly over DNS.
+This means the pod needs to have appropriate network policies allowing it to talk to exactly the components required to execute its desired functionality.
+[This has also implications for Gardener extension](#implications-for-gardener-extensions) that need to deploy additional components into the `Shoot's` control plane.
+
+### Network Policies for Control Plane Components
+
+This section highlights a selection of network policies that exist in the `Shoot` namespace in the `Seed` cluster.
+In general, the control plane components serve different purposes and thus need access to different pods and network ranges.
+
+In contrast to other network policies, the policy `allow-to-shoot-networks` is tailored to the individual `Shoot` cluster, 
+because it is based on the network configuration in the `Shoot` manifest.
+It allows pods with the label `networking.gardener.cloud/to-shoot-networks=allowed` to access pods in the `Shoot` pod, 
+service and node CIDR range. This is used by the `Shoot` API Server and the prometheus pods to communicate over VPN/proxy with pods in the Shoot cluster.
+
+The policy `allow-to-blocked-cidrs` allows pods with the label `networking.gardener.cloud/to-blocked-cidrs=allowed` to access IPs that are explicitly blocked for all control planes in a `Seed`cluster (configurable via `seed.Spec.networks.blockCIDRS`). This is used for instance to block the cloud provider's metadata service.
+
+Another network policy to be highlighted is `allow-to-seed-apiserver`.
+Some components need access to the `Seed` API Server. This can be allowed by labeling the pod with `networking.gardener.cloud/to-seed-apiserver=allowed`.
+This policy allows exactly the IPs of the `kube-apiserver` of the `Seed`.
+While all other policies have a static set of permissions (do not change during the lifecycle of the `Shoot`), the policy `allow-to-seed-apiserver` is reconciled to reflect the endpoints in the `default` namespace.
+This is required because endpoint IPs are not stable (think of scaling the `Seed` API Server pods or hibernating the `Seed` cluster in a local development environment).
```

Comment:
```suggestion
This is required because endpoint IPs are not necessarily stable (think of scaling the `Seed` API Server pods or hibernating the `Seed` cluster in a local development environment).
```

Usually, the IPs are stable, e.g., in mastered clusters or when the API server is backed by a load balancer. They are not necessarily stable when the seed is a shoot as well because of the "special" network topology.